src/Lean/PrettyPrinter/Delaborator/Basic.lean

```diff
@@ -232,12 +232,21 @@ where
     stx := stx
   }
 
-def addOmissionInfo (pos : Pos) (stx : Syntax) (e : Expr) : DelabM Unit := do
+inductive OmissionReason
+  | deep
+  | proof
+
+def OmissionReason.toString : OmissionReason → String
+  | deep => "This `⋯` denotes a term omitted due to its depth (see option `pp.deepTerms`)."
+  | proof => "This `⋯` denotes an omitted proof (see option `pp.proofs`)."
```

Comment:
Even more brief: "Term omitted due to its depth (see option `pp.deepTerms`)" 
I think it should be obvious from the symbol that `⋯` is some kind of omitted term.